zcash_client_backend/src/data_api.rs

```diff
@@ -2056,9 +2060,8 @@ impl AccountBirthday {
     ///    exist no unscanned ranges between the wallet's birthday height and the provided
     ///    `recover_until` height, exclusive.
     ///
-    /// This API is intended primarily to be used in testing contexts; under normal circumstances,
+    /// This API is intended primarily to be used in testing contexts and deserialization; under normal circumstances,
```

Comment:
Why is this API needed for deserialization, rather than `AccountBirthday::from_treestate`?